src/io/p5.Table.js

```diff
@@ -639,6 +639,35 @@ p5.Table.prototype.addColumn = function(title) {
  *
  *  @method  getColumnCount
  *  @return {Integer} Number of columns in this table
+ * @example
+ * <div>
+ * <code>
+ * // given the cvs file "blobs.csv" in /assets directory
+ * // ID, Name, Flavor, Shape, Color
+ * // Blob1, Blobby, Sweet, Blob, Pink
+ * // Blob2, Saddy, Savory, Blob, Blue
+ *
+ * var table;
+ *
+ * function preload() {
+ *   table = loadTable('assets/blobs.csv');
+ * }
+ *
+ * function setup() {
+ *   createCanvas(200, 100);
+ *   textAlign(CENTER);
+ *   background(255);
+ * }
+ *
+ * function draw() {
+ *   text(
+    'There are ' + table.getColumnCount() + ' columns in the table.',
+    100,
+    50
```

Comment:
this is a little hard to read, can we do it in one line like line 703?
if that's too long, I'd suggest pulling out the string like:
```javascript
var str = 'There are ' + table.getColumnCount() + ' columns in the table.';
text(str, 100, 50);
```